server/src/main/java/io/unitycatalog/server/persist/dao/VolumeInfoDAO.java

```diff
@@ -45,8 +54,11 @@ public VolumeInfo toVolumeInfo() {
         .name(getName())
         .comment(comment)
         .storageLocation(FileUtils.convertRelativePathToURI(storageLocation))
+        .owner(owner)
         .createdAt(createdAt.getTime())
+        .createdBy(createdBy)
         .updatedAt(updatedAt.getTime())
+        .updatedBy(updatedBy)
         .volumeType(VolumeType.valueOf(volumeType));
   }
 
```

Comment:
The `from` method below needs to be updated to include owner, createdBy, and updatedBy

Reply:
Fixing.